src/build.d

```diff
@@ -84,29 +83,23 @@ Command-line parameters
         return;
     }
 
-    if (res.helpWanted)
-        return showHelp;
-
     // parse arguments
     args.popFront;
     args2Environment(args);
+    parseEnvironment;
+    processEnvironment;
+
+    if (res.helpWanted)
```

Comment:
In a future improvement.  The help can use a global list of dependencies to pull information from.  However, those dependencies can't be created until the environment is initialized.  So we move the help to be printed after initializing the environment.